packages/slate-editor/src/components/EditorBlock/EditorBlock.tsx

```diff
@@ -82,24 +82,35 @@ export const EditorBlock = forwardRef<HTMLDivElement, Props>(function (
 
     const [menuOpen, setMenuOpen] = useState(true);
     const [container, setContainer] = useState<HTMLDivElement | null>(null);
-    const openMenu = useCallback(() => setMenuOpen(true), [setMenuOpen]);
-    const closeMenu = useCallback(() => setMenuOpen(false), [setMenuOpen]);
+    const closeMenu = useCallback(() => setMenuOpen(false), []);
 
-    const handleClick = useCallback(
-        function () {
-            openMenu();
+    const handleBlockClick = useCallback(
+        function (event) {
```

Comment:
I think it is better to add type annotations because useCallback implicitly makes arguments to be any

Reply:
Fixed. Thanks!